client-app/src/css/AddProgramPage.css

```diff
@@ -43,5 +76,38 @@
 }
 
 .back-button {
```

Comment:
I would center the back button like you did for the save and clear, I think it would make it more visually appealing